hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -225,6 +312,31 @@ export const ReturnedTable = () => {
 export const PendingTables = () => {
     const dispatch = useDispatch();
     const orders = useSelector(pendingOrderSelectors.selectAll);
+    let ready_orders = [];
```

Comment:
I suggest abstracting the logic into a separate function, such as `helpers.ts`, to avoid cluttering the file with excessive code. Ideally you would just do one function call for sorting the orders here, and in other parts of the code.